data/resources/ui/content-chat-info-window.ui

```diff
@@ -12,58 +13,109 @@
+                                    <property name="justify">center</property>
+                                    <style>
+                                      <class name="title-1"/>
+                                    </style>
+                                  </object>
+                                </child>
+                                <child>
+                                  <object class="GtkInscription" id="subtitle_label">
+                                    <property name="text-overflow">ellipsize-end</property>
+                                    <property name="xalign">0.5</property>
+                                  </object>
+                                </child>
+                                <child>
+                                  <object class="GtkListBox" id="info_list">
+                                    <property name="selection-mode">none</property>
+                                    <style>
+                                      <class name="boxed-list"/>
+                                    </style>
+                                  </object>
+                                </child>
+                              </object>
+                            </property>
+                          </object>
+                        </property>
                       </object>
-                    </child>
-                    <child>
-                      <object class="GtkLabel" id="name_label">
-                        <property name="wrap">True</property>
-                        <property name="justify">center</property>
-                        <style>
-                          <class name="title-1"/>
-                        </style>
-                      </object>
-                    </child>
-                    <child>
-                      <object class="GtkInscription" id="subtitle_label">
-                        <property name="text-overflow">ellipsize-middle</property>
-                        <property name="xalign">0.5</property>
-                      </object>
-                    </child>
-                    <child>
-                      <object class="GtkListBox" id="info_list">
-                        <property name="selection-mode">none</property>
-                        <style>
-                          <class name="boxed-list"/>
-                        </style>
+                    </property>
+                  </object>
+                </child>
+                <child>
+                  <object class="AdwViewStackPage" id="members_page">
+                    <property name="name">members</property>
+                    <property name="title" translatable="yes">Members</property>
+                    <property name="icon-name">avatar-default-symbolic</property>
+                    <property name="visible">false</property>
+                    <property name="child">
+                      <object class="GtkScrolledWindow">
```

Comment:
Since this list will probably get more features in the future, I suggest putting all of this in a separate custom widget and also move the logic in it.